presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
```

Comment:
The wording in highlights section are not consistent. Should we use similar guidelines as the normal release notes like "improve..." "add ..." etc?

Reply:
Sure I can make them match.